EFCore3.Data/Repositories/UserRepository.cs

```diff
@@ -0,0 +1,47 @@
+﻿using DeliveryServiceModel;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace EFCore5.Data
+{
+    public class UserRepository : IRepository<User>
+    {
+        private DataContext db;
+
+        public UserRepository(DataContext context)
+        {
+            db = context;
+        }
+
+        public IEnumerable<User> GetAll(Predicate<string> predicate)
```

Comment:
seems like GetAll is not best name cause it is not get all
it is get by filter
up to u